From 911423bb889f427af64e192bb3a3f972ddb2d448 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Tue, 16 Aug 2016 10:57:44 -0400
Subject: [PATCH 0734/1722] drm/amd/dal: Don't treat warnings as errors

Due to mismatched gcc versions and people not always
following clean coding guidelines this causes too many
issues. Removing it.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/Makefile | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/Makefile b/drivers/gpu/drm/amd/dal/Makefile
index b84a2db..2ef2128 100644
--- a/drivers/gpu/drm/amd/dal/Makefile
+++ b/drivers/gpu/drm/amd/dal/Makefile
@@ -5,8 +5,6 @@
 
 AMDDALPATH = $(RELATIVE_AMD_DAL_PATH)
 
-subdir-ccflags-y += -Werror
-
 subdir-ccflags-y += -I$(AMDDALPATH)/ -I$(AMDDALPATH)/include
 
 subdir-ccflags-y += -I$(FULL_AMD_DAL_PATH)/dc/inc/
-- 
2.7.4

